src/command.rs

```diff
@@ -73,6 +74,27 @@ impl Command {
         }
         state.loop_counter += 1;
       }
+      Self::Open(path) => {
+        let default_command = {
+          if cfg!(target_os = "macos") {
+            Some("open")
+          } else if cfg!(target_os = "linux") {
+            Some("xdg-open")
+          } else {
+            None
+          }
+        };
+
+        let command = env::var("DEGENERATE_OPEN_COMMAND").unwrap_or(
+          default_command
+            .ok_or("Default command for 'open' not set for target os.")?
+            .to_string(),
+        );
```

Comment:
Maybe something like this?

```suggestion
        let command = env::var("DEGENERATE_OPEN_COMMAND").unwrap_or(
          if cfg!(target_os = "macos") {
            "open"
          } else if cfg!(target_os = "linux") {
            "xdg-open"
          } else {
            return Err("Please supply an open comment by setting the `DEGENERATE_OPEN_COMMAND` environment variable");
          }
        );
```